app/handlers/objectTracking.py

```diff
@@ -108,9 +108,17 @@ def run(self):
         if os.path.exists(db_path):  # If results database already exists,
             os.remove(db_path)  # then remove it--it'll be recreated.
 
+        fbttf_call = ["feature-based-tracking", tracking_path, "--tf", "--database-filename", db_path]
+        fbtgf_call = ["feature-based-tracking", tracking_path, "--gf", "--database-filename", db_path]
+
+        mask_filename = os.path.join(get_project_path(self.identifier), "mask.jpg")
```

Comment:
Why be able to upload mask with any extension, but only check for jpg here? I think the upload should save mask filename in config.cfg, and add a function to get the filename of the mask. If we're trying to get this in for Needham analysis, you can ignore this and ship it for now though.